mantidproject/mantid
# Add MDNorm* benchmarks to release notes.

Description of work.

 This adds release notes showing performance improvements from pull requests #19451, #19462 and #19530.

**To test:**

Build the HTML documentation and check the new content and formatting.



This addresses a comment on our slack channel.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
